src/main/java/com/minecolonies/network/messages/BuildToolPlaceMessage.java

```diff
@@ -197,15 +199,15 @@ private static void handleHut(@NotNull World world, @NotNull EntityPlayer player
      * @param rotation   The number of times the decoration is rotated.
      * @param buildPos   The location the decoration will be built.
      */
-    private static void handleDecoration(@NotNull World world, @NotNull EntityPlayer player, String decoration, String style, int rotation, @NotNull BlockPos buildPos)
+    private static void handleDecoration(@NotNull final World world, @NotNull final EntityPlayer player, final String decoration, final String style, final int rotation, @NotNull final BlockPos buildPos)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 203 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)